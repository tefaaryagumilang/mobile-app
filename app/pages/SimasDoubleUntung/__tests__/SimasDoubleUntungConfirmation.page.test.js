import React from 'react';
import renderer from 'react-test-renderer';
import SimasDoubleUntungConfirmationPage from '../SimasDoubleUntungConfirmation.page';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';

const store = createStore(() => ({}));
const DecoratedForm = reduxForm({form: 'testForm'})(SimasDoubleUntungConfirmationPage);

describe('Simas Double Untung Confirmation Page', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Provider store={store}>
        <DecoratedForm />
      </Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});