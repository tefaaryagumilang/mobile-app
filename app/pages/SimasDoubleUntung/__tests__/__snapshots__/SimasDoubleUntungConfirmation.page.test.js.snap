// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Simas Double Untung Confirmation Page renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F1F3F6",
      "flex": 1,
    }
  }
>
  <KeyboardAwareScrollView
    contentContainerStyle={
      Object {
        "justifyContent": "space-between",
      }
    }
    keyboardShouldPersistTaps="handled"
  >
    <View
      style={
        Object {
          "flex": 1,
          "paddingTop": 0,
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#FA3F70",
            "height": 90,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 15,
            "marginHorizontal": 20,
            "marginTop": -70,
            "padding": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#2F476D",
              "fontSize": 18,
              "fontWeight": "700",
            }
          }
        >
          Nominal Penempatan
        </Text>
        <View
          style={
            Object {
              "alignItems": "center",
              "borderColor": "#ccc",
              "borderRadius": 10,
              "borderWidth": 1,
              "flexDirection": "row",
              "marginTop": 10,
              "padding": 15,
            }
          }
        >
          <Image
            source="icon_rp.png"
            style={
              Object {
                "height": 30,
                "width": 30,
              }
            }
          />
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontSize": 24,
                "fontWeight": "700",
                "marginLeft": 20,
              }
            }
          >
            --
          </Text>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "marginHorizontal": 40,
              "marginTop": 20,
            },
            Object {
              "color": "#2F476D",
              "fontSize": 18,
              "fontWeight": "700",
            },
          ]
        }
      >
        Ringkasan
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderRadius": 15,
            "marginBottom": 20,
            "marginHorizontal": 20,
            "marginTop": 10,
            "padding": 10,
          }
        }
      >
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Email
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Rekening sumber dana
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            
            

            
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Periode lock dana
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            
             months
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Tanggal mulai
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Tanggal jatuh tempo
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            
          </Text>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "paddingHorizontal": 10,
              "paddingVertical": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "fontFamily": "roboto",
              }
            }
          >
            Manfaat setara bunga
          </Text>
          <Text
            style={
              Object {
                "color": "#2F476D",
                "flex": 1,
                "textAlign": "right",
              }
            }
          >
            -
            % pa
          </Text>
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#FFFFFF",
            "borderRadius": 15,
            "flexDirection": "row",
            "marginBottom": 20,
            "marginHorizontal": 20,
            "overflow": "hidden",
            "paddingHorizontal": 20,
            "paddingVertical": 10,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": undefined,
                "fontSize": 20,
              },
              Object {
                "color": "#2F476D",
              },
              Object {
                "fontFamily": "simas",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          î¥œ
        </Text>
        <Text
          style={
            Object {
              "color": "#2F476D",
              "flex": 1,
              "fontSize": 12,
              "marginLeft": 10,
            }
          }
        >
          Manfaat setara bunga
           
          -
          % pa
          . 
          terdiri dari Cashback, Bunga Tabungan, dan Pajak Hadiah.
        </Text>
      </View>
    </View>
  </KeyboardAwareScrollView>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "paddingHorizontal": 20,
        "paddingVertical": 20,
      }
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FA3F70",
              "borderRadius": 30,
              "flexDirection": "row",
              "height": 47,
              "justifyContent": "center",
              "paddingHorizontal": 15,
            },
            undefined,
            Object {},
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
            }
          }
        >
          BERIKUTNYA
        </Text>
      </View>
    </View>
  </View>
</View>
`;
